assets/out

```diff
@@ -79,7 +80,11 @@ tag() {
 }
 
 push_src_and_tags() {
-  git push --tags push-target HEAD:refs/heads/$branch
+  if [ "$force" = "true" ]; then
+    git push --tags push-target HEAD:refs/heads/$branch --force
+  else
+    git push --tags push-target HEAD:refs/heads/$branch
+  fi
 }
 
 push_tags() {
```

Comment:
may also be worth updating this for the tag-only push case.

maybe just extract a `$forceflag` var that's either ` --force` or empty, and interpolate it into the arg string for both `push`es?